src/components/encoding-pane/range-filter-shelf.tsx

```diff
@@ -199,12 +197,12 @@ export class RangeFilterShelfBase extends React.Component<RangeFilterShelfProps,
   }
 
   // TODO: https://github.com/vega/voyager/issues/443: use the time formatter Vega derives from D3
-  private formatTime = (value: number) => {
+  private formatTime = (value: number) : React.ReactText => {
```

Comment:
`(value: number): React.ReactText`  (no space after `)`)

Also wonder why it has to be `React.ReactText`?  What's wrong with just being a string?